modules/sandboxed-containers-uninstalling-kata-runtime.adoc

```diff
@@ -0,0 +1,18 @@
+//Module included in the following assemblies:
+//
+// *disabling-sandboxed-container-workloads.adoc
+
+[id="sandboxed-containers-uninstalling-kata-runtime_{context}"]
+
+= Uninstalling Kata Runtime
+
+This section describes how to remove and uninstall the `kata` runtime and all
+its related resources (for example, CRI-O config, RuntimeClass, and so on) from
```

Comment:
Consider removing this parenthetical entirely. If it's required, suggested edit:
> related resources, such CRI-O config and `RuntimeClass`, from